drape_frontend/visual_params.cpp

```diff
@@ -70,7 +70,7 @@ string const & VisualParams::GetResourcePostfix(double visualScale, bool isYotaD
 
   // Looking for the nearest available scale.
   int postfixIndex = -1;
-  double minValue = numeric_limits<float>::max();
+  double minValue = numeric_limits<double>::max();
```

Comment:
@mgsergio proposing to use 'auto' keyword when variable's type is obvious to reviewer and compiler. It's up to you whether to change this or leave as is.